usaspending_api/references/management/commands/load_psc.py

```diff
@@ -1,81 +1,134 @@
+import datetime
+import logging
+from typing import Optional, Union, List, Tuple
+from urllib.error import HTTPError
+
+import pandas as pd
 from django.core.management.base import BaseCommand
+
 from usaspending_api.references.models import PSC
-import os
-import logging
-from openpyxl import load_workbook
 
 
 class Command(BaseCommand):
     help = "Loads program information obtained from Excel file on https://www.acquisition.gov/PSC_Manual"
 
     logger = logging.getLogger("script")
-    default_directory = os.path.normpath("usaspending_api/references/management/commands/")
-    default_filepath = os.path.join(default_directory, "PSC_Data_June_2019_Edition_FINAL_6-20-19+DRW.xlsx")
+    default_filepath = "https://www.acquisition.gov/sites/default/files/manual/PSC%20April%202024.xlsx"
```

Comment:
Do we want to host this file in a storage bucket instead to ensure that it is available?

Reply:
I think that's a good idea. Plus, if we self-host it we know nothing is changing unexpectedly. We would just need to remember to go download and then upload the updated file whenever it's updated.

If we do decide to self-host this file, any thoughts on using a more standardized filename instead of one with the month and year in it? I see that we can use the `--path` flag to change which spreadsheet is being loaded, but it could be convenient to have a default value of `latest_psc.xlsx`. That way if/when we uploaded a new file we could just name it `latest_psc.xlsx` and not need to worry about updating `self.default_filepath` or updating the Jenkins job to specify a new `--path` value. I'm not tied to this idea, but it might make long-term maintenance of this command easier.

Reply:
> any thoughts on using a more standardized filename instead of one with the month and year in it?
Not entirely against the idea, but we would want to maintain an easy way to determine exactly what file we uploaded previously. That could be tags on the S3 object or a comment in the code.